BundleURL.swift

```diff
@@ -0,0 +1,34 @@
+import CoreML
+
+@available(macOS 10.13, iOS 11.0, tvOS 11.0, watchOS 4.0, *)
+//model name : "FindFour.mlmodelc"
+struct BundleURL{
+    static func bundleUrl(modelName: String) -> URL?{
+        var modelcFile: URL
+        if let documentDirectory =
```

Comment:
you can use a `guard` here also since we're always going to need this directory and if we can get it we can't compile